javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,44 @@
+let userName = [];
+let myTodo = [];
+function getReply(command) {
+  if (command.includes("my name is")) {
+    let commandStr = command.split(" ");
+    let activeName = commandStr[commandStr.length - 1];
+
+    if (userName.includes(activeName)) {
+      console.log(" Yes I know");
+      return;
```

Comment:
Since you're both returning and console.logging you can do:
```js
return console.log("Yes I know");
```
It saves you a line